src/Client/States/SelectedColumns.fs

```diff
@@ -1,4 +1,4 @@
-module Shared.DTOs.SelectedColumnsModalDto
+module SelectedColumns
```

Comment:
This is not a state either. Move to specific module in `Types.fs`

Reply:
I grouped all elements that were used by SeletiveTemplateFromDB.fs and SelectiveImporModal.fs into the ModalElements.fs.
Origininally I rebuild the whole template selection as a modal but have undone it.

Reply:
fixed